rero_ils/modules/documents/views.py

```diff
@@ -657,10 +657,11 @@ def document_main_type(record, translate=True):
     :param translate: translate document type
     :return: document main type
     """
-    doc_type = record['type'][0]['main_type']
-    if translate:
-        doc_type = _(doc_type)
-    return doc_type
+    if 'type' in record:
+        doc_type = record['type'][0]['main_type']
+        if translate:
+            doc_type = _(doc_type)
+        return doc_type
```

Comment:
COSMETIC: Could be more synthetic 
```
if 'type' not in record:
    return None
doc_type = record['type'][0]['main_type']
return _(doc_type) if translate else doc_type
```
But, even me, I don't like ternary instruction pattern with python